File: app/src/main/java/org/bluechat/blueninemenmoris/model/HumanPlayer.java
package org.bluechat.blueninemenmoris.model;

public class HumanPlayer extends Player {
		
	public HumanPlayer(String name, Token player, int numPiecesPerPlayer) throws GameException {
		super(player, numPiecesPerPlayer);
		this.name = name;
	}

	@Override
	public boolean isAI() {
		return false;
	}
}

File: app/src/main/java/org/bluechat/blueninemenmoris/model/HumanPlayer.kt
package org.bluechat.blueninemenmoris.model

class HumanPlayer @Throws(GameException::class)
constructor(name: String, player: Token, numPiecesPerPlayer: Int) : Player(player, numPiecesPerPlayer) {

    init {
        this.name = name
    }

    override val isAI: Boolean
        get() = false
}
